BotProject/Templates/CSharp/Scripts/build_runtime.ps1

```diff
@@ -0,0 +1,25 @@
+Param(
+	[object] $config,
+	[string] $customSettingFolder,
+    [string] $luisAuthroingKey,
+    [SecureString] $appPassword,
+    [string] $projFolder = $(Join-Path $(Get-Location) BotProject CSharp)
+)
+
+if ($PSVersionTable.PSVersion.Major -lt 6){
+	Write-Host "! Powershell 6 is required, current version is $($PSVersionTable.PSVersion.Major), please refer following documents for help."
+	Write-Host "For Windows - https://docs.microsoft.com/en-us/powershell/scripting/install/installing-powershell-core-on-windows?view=powershell-6"
+	Write-Host "For Mac - https://docs.microsoft.com/en-us/powershell/scripting/install/installing-powershell-core-on-macos?view=powershell-6"
+	Break
+}
+
+if ((dotnet --version) -lt 3) {
+	Write-Host "! dotnet core 3.0 is required, please refer following documents for help."
+	Write-Host "https://dotnet.microsoft.com/download/dotnet-core/3.0"
+	Break
+}
+
+# Init user secret id
+dotnet user-secrets init
```

Comment:
do we really need dotnet user-secrets here? I don't see where we use that. you are passing the sensitive info throught the command line, correct?

Reply:
yes, dotnet core need user-secret to specify the runtime.
yes, sensitive settings will pass through command line